openff/benchmark/cli.py

```diff
@@ -486,6 +488,155 @@ def plots(input_path, ref_method, output_directory):
     draw.plot_compare_ffs(input_path, ref_method, output_directory)
 
 
+@cli.group()
+def schrodinger():
+    """Generate OPLS parameters and optimize using Schrodinger tools.
+    """
+    pass
+
+@schrodinger.command()
+@click.option('--schrodinger-path', default=None, help="Path to schrodinger binaries. Defaults to $SCHRODINGER.")
+@click.option('--host', default='localhost', help="Host name as specified in SCHRODINGER_HOSTS.")
+@click.option('--max-jobs', default=None, help="Maximal number of subjobs. Defaults to no limit on subjobs.")
+@click.option('--opls-dir', default=None, help="Path to the custom OPLSDIR. If it is not given, a default path ($HOME/.schrodinger/opls_dir) will be used.")
+@click.option('--recursive', is_flag=True, help="Recursively traverse directories for SDF files to submit")
+@click.option('-o', '--output-path', default='7-schrodinger_ffb', help="Path where output files are written to.")
+@click.argument('input-path', nargs=-1)
+def ffbuilder(input_path, schrodinger_path, host, max_jobs, opls_dir, recursive, output_path):
+    """Build OPLS3e force field parameters using molecules from INPUT_PATH.
+
+    INPUT_PATH may be any number of single SDF files, or any number of directories containing SDF files to submit.
+
+    To recurse directory INPUT_PATHs, use the `--recursive` flag.
+
+    """
+    from .schrodinger import ffbuilder
+
+    if schrodinger_path is None:
+        schrodinger_path = os.getenv('SCHRODINGER')
+    if not os.path.isdir(schrodinger_path):
+        raise ValueError("A valid SCHRODINGER path is not given.")
+    if opls_dir is None:
+        warnings.warn("The specified OPLS custom parameter path is not given. A default path is used.")
+        opls_dir = os.path.join(os.getenv('HOME'), '.schrodinger', 'opls_dir')
+    if not os.path.isdir(opls_dir):
+        raise ValueError("The specified OPLS custom parameter path is not found. Exiting.")
```

Comment:
Can we create the directory if it doesn't exist, and just warn that it is being created? This would be non-destructive and handles the case where a user doesn't have an `opls_dir` at all.

Reply:
Thanks for the suggestion. This is now done.